preprocessors/caption-recognition/caption.py

```diff
@@ -87,7 +87,8 @@ def captions():
     except jsonschema.exceptions.ValidationError as e:
         logging.error(e)
         return jsonify("Invalid Preprocessor JSON format"), 500
-    logging.debug(data)
+    logging.debug("Caption Recoginition Response")
```

Comment:
If you're interested in having a clear indication of the preprocessor on each line (beyond what docker will do anyway in compose mode), you can get this easily with [a named logger](https://docs.python.org/3/library/logging.html#logging.getLogger). It would only require making calls through the logger object rather than the `logging` module, although it may not be crucial for smaller preprocessors like this.